tests/tck/conftest.py

```diff
@@ -203,6 +202,17 @@ def try_to_execute_query(query, graph_spaces, session, request):
     for stmt in ngql.split(';'):
         exec_query(request, stmt, session, graph_spaces, True)
 
+@when(parse("clone a new space according to current space:\n{new_space}"))
```

Comment:
I don’t think there needs a new_space parameter here. Random generate the space name.

Reply:
👌